dist/info/mu_libretro.info

```diff
@@ -11,26 +12,10 @@ display_version = "v1.1.0"
 supports_no_game = "true"
 
 # BIOS/Firmware
-firmware_count = 5
-
-firmware0_desc = "palmos40-en-m500.rom (Palm OS 4.0)"
-firmware0_path = "palmos40-en-m500.rom"
-firmware0_opt = "true"
+firmware_count = 1
 
 firmware1_desc = "palmos41-en-m515.rom (Palm OS 4.1)"
 firmware1_path = "palmos41-en-m515.rom"
 firmware1_opt = "true"
```

Comment:
Would have `firmware1_opt = "false"` as the core will not function if the ROM is missing.